agent_app_sample_code/00_global_config.py

```diff
@@ -0,0 +1,200 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Global configuration
+# MAGIC
+# MAGIC This notebook is meant to be changed once throughout the lifetime of your Agent.
+# MAGIC
+# MAGIC At a high-level it defines the Unity Catalog locations where the agent, and associated resources, will be stored. Ideally you shouldn't have to change the values in this notebook once you've started building your agent.
+
+# COMMAND ----------
+
+# If you run this notebook directly, you need these dependencies:
+# %pip install -U -qqqq databricks-sdk mlflow mlflow-skinny
+# dbutils.library.restartPython()
+
+# COMMAND ----------
+
+# By default, will use the current user name to create a unique UC catalog/schema & vector search endpoint
+user_email = spark.sql("SELECT current_user() as username").collect()[0].username
+user_name = user_email.split("@")[0].replace(".", "").lower()[:35]
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Agent configuration
+# MAGIC
+# MAGIC Important: These notebooks only work on Single User clusters running DBR/MLR 14.3+.
+# MAGIC
+# MAGIC To begin with, we simply need to configure the following:
+# MAGIC 1. `AGENT_NAME`: The name of the Agent.  Used to name the app's Unity Catalog model and is prepended to the output Delta Tables + Vector Indexes
```

Comment:
`AGENT_NAME` is different from `UC_MODEL_NAME`?

Reply:
`UC_MODEL_NAME` is the 3 level name composed of `UC_CATALOG}.{UC_SCHEMA}.{AGENT_NAME}`